pvlib/bifacial/loss_models.py

```diff
@@ -0,0 +1,175 @@
+import numpy as np
+import pandas as pd
+
+from typing import Literal
+
+
+def power_mismatch_deline(
+    rmad,
+    model: Literal[
+        "fixed-tilt", "single-axis-tracking"
+    ] = "single-axis-tracking",
+    fillfactor: float = None,
+):
+    r"""
+    Estimate DC power loss due to irradiance non-uniformity.
+
+    This model is described for bifacial modules in [1]_, where the backside
+    irradiance is less uniform due to mounting and site conditions.
+
+    Depending on the mounting type, the power loss is estimated with either
+    equation (11) or (12) of [1]_. Passing a custom polynomial is also valid.
+
+    Use ``fillfactor`` to account for different fill factors between the
+    trained model and the module of interest.
+    For example, if the fill factor of the module of interest is
+    0.65, then set ``fillfactor=0.65``.
+
+    .. versionadded:: 0.11.0
+
+    Parameters
+    ----------
+    rmad : numeric
+        The Relative Mean Absolute Difference of the cell-by-cell total
+        irradiance. [Unitless]
+        Check out the *Notes* section for the equation to calculate it from the
+        bifaciality and the front and back irradiances.
+
+    model : str, numpy.polynomial.polynomial.Polynomial or list, default ``"single-axis-tracking"``
+        The model coefficients to use.
+        If a string, it must be one of the following:
+
+        * ``"fixed-tilt"``: Eq. (11) of [1]_.
+        * ``"single-axis-tracking"``: Eq. (12) of [1]_.
```

Comment:
The reference actually recommends using the fixed-tilt coefficients for both fixed and tracking (see the discussion around Table 4 in [1]):

> It is the authors' suggestion that Fit # 2, which is based on calculated mean average difference in Gtotal, be used for all bifacial system scenarios because it provides the best overall annual match to the full cell-level simulation.

With that in mind, I don't think we should provide the "single-axis-tracking" option as the default or even as a suggestion.  I suggest we change this `model` parameter to only accept numeric coefficients, which default to the fixed-tilt (Eq 11) values.

Reply:
That's okay from my side 👍